_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -206,6 +206,35 @@ You will most likely not need to specify any parameters except for `location`. F
 
 You will most likely not need to specify any parameters except for `bucket` and `base_path`. For allowed request parameters, see [Register or update snapshot repository API](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/).
 
+### Microsoft azure
+
+1. To use a Microsoft Azure Blob Storage as a snapshot repository, install the `repository-azure` plugin on all nodes:
+
+   ```bash
+   sudo ./bin/opensearch-plugin install repository-azure
+   ```
+
+1. (Optional) If you're using token credential instead of SAS token or an account key:
+
+   1. Define your azure storage account name in secure setting:
+
+      ```bash
+      sudo ./bin/opensearch-keystore add azure.client.default.account
```

Comment:
```suggestion
  ./bin/opensearch-keystore add azure.client.default.account
  ./bin/opensearch-keystore add azure.client.default.key
```

Reply:
I believe either `sas_key` or `key` has to be used

Reply:
yup, just adding one more dot point when configuring sas token to account key